### PR TITLE
Restriction implemented for targeted run events

### LINKED ISSUE #2941: Run restrictions (Peace in our Time) don't prevent run events with specific targets
Hit the issue when someone could use Indexing after playing Peace in our Time. I don't know if there are other conditions that can cause the issue but ":req (req (not (:<hq/rd/archives> (get-in runner [:register :cannot-run-on-server]))))" seems a simple general purpose fix to the issue for all affected cards but Demolition Run. If you give me the go-ahead, I can open a pull request for it.

Affected cards:
Account Siphon
CBI Raid
Code Siphon
Data Breach
Deep Data Mining
Demolition Run
Escher
Fear The Masses
Feint
Immolation Script
Indexing
Information Sifting
Legwork
Möbius
Political Graffiti
Retrieval Run
Rip Deal
Showing Off
The Maker's Eye
Vamp
Wanton Destruction

### PR DESCRIPTION
**Pull request by mcyalcin**
Fixes #2941

### CODE REVIEW

File: src/clj/game/cards/events.clj [hunk 1]
```diff
@@ -16,7 +16,8 @@
 
 (def cards-events
   {"Account Siphon"
-   {:effect (effect (run :hq {:req (req (= target :hq))
+   {:req (req rd-runnable)
```
Comment (nealterrell) [inline src/clj/game/cards/events.clj]: I think you mean hq-runnable.

**Comment by nealterrell** [comment]
Sorry for the delay. One minor change and then we can merge this!

**Comment by mcyalcin** [comment]
Sorry for the mistake, these always happen with mass repetitions. Thanks for noticing. Corrected and resolved conflicts.

**Comment by nealterrell** [comment]
👍

**Pull request merged**